### PR TITLE
COMMERCE-9814 Add functionality for more exact search with double quote search

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9814

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java [hunk 1]
```diff
@@ -378,6 +380,15 @@ public void postProcessSearchQuery(
 				else {
 					searchQuery.add(booleanQuery, BooleanClauseOccur.SHOULD);
 				}
+
+				if (keywords.startsWith(StringPool.QUOTE) &&
+					keywords.endsWith(StringPool.QUOTE) &&
+					(keywords.length() > 1)) {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java:386]: I can't find examples of this pattern in the code base, are you sure this is the right way to make an exact search?

**Review by andrea-ale-sbarra — COMMENTED** [review]
@alessiorendina @luca-pellizzon can you have a look at this too?

**Comment by andrea-ale-sbarra** [comment]
https://liferay.slack.com/archives/CKY6GP7BL/p1676652933975769

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java (hunk 1 above)
Comment (dsitu) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java:386]: @andrea-ale-sbarra yeah this is not a existing pattern. Hmm, if this is too hacky we might be able to do something with additional-type-mappings.

Comment (luca-pellizzon) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/CPDefinitionIndexer.java:386]: I think we need to find a better solution

**Pull request closed without merging**